CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 Mapbox welcomes participation and contributions from everyone.
 
 # main
-
+## Bug fixes 🐞
```

Comment:
Also - as we're porting this to v10.9 branch - we should then create that header now and remove this entry from `#main`

Reply:
I'll move it in the v10.9 branch probably

Reply:
then we might forget about it in `main` branch and accidently include it in v10.10.0-beta.1 that will grab changelog from `#main` section

Reply:
yeah, agree to add 10.9.0 change log section now, so we can just cherry-pick this to the release branch, and our release script should be able to pick it up automatically.

Reply:
Okay